vocabularies/Org.OData.Validation.V1.xml

```diff
@@ -117,6 +117,9 @@
           <Annotation Term="Core.IsLanguageDependent" />
           <Annotation Term="Core.Description" String="Human-readable message that can be shown to end users if the constraint is not fulfilled" />
         </Property>
+        <Property Name="MessageSeverity" Type="Core.MessageSeverity" Nullable="true">

```

Comment:
In Core.MessageType we only call it `severity`, so `Severity` here to match case would also be fine. Either way.